lib/lightning_web/live/job_live/inspector_form_component.ex

```diff
@@ -31,8 +31,11 @@ defmodule LightningWeb.JobLive.InspectorFormComponent do
 
       :new ->
         case Jobs.create_job(
-               job_params
-               |> Map.put("project_id", socket.assigns.job.project_id)
+               Map.put(
```

Comment:
same here --> is this part of the PR? if not please remove

Reply:
Yeah this was the first attempt of this work. After that things have changed / moved a lot and I forgot to restore the file to it's original format. But nothing too serious, will change it to a pipe.